app/serializers/rest/v1/instance_serializer.rb

```diff
@@ -83,6 +83,8 @@ def configuration
       },
 
       media_attachments: {
+        supported_mime_types: MediaAttachment::IMAGE_MIME_TYPES + MediaAttachment::VIDEO_MIME_TYPES + MediaAttachment::AUDIO_MIME_TYPES,
```

Comment:
duplicated line with 88 below? `MediaAttachment.supported_mime_types` is this :)